aasemble/django/apps/buildsvc/views.py

```diff
@@ -92,6 +92,17 @@ def builds(request):
 
 
 @login_required
+def rebuild(request, source_id):
+    ps = PackageSource.objects.get(pk=source_id)
```

Comment:
You may want to check if the user has access to this package source. Currently the user can rebuild any package source by just entering the ID of the package source. It is fairly easy to guess the ID of a package source as it is an integer.